io.github.mhogomchungu.sirikali.json

```diff
@@ -0,0 +1,48 @@
+{
+  "app-id" : "io.github.mhogomchungu.sirikali",
+  "runtime" : "org.kde.Platform",
+  "runtime-version" : "6.7",
+  "sdk" : "org.kde.Sdk",
+  "command" : "io.github.mhogomchungu.sirikali",
+  "finish-args" : [
+    "--share=ipc",
+    "--socket=wayland",
+    "--socket=fallback-x11",
+    "--filesystem=xdg-documents",
+    "--share=network",
+    "--device=dri",
+    "--talk-name=org.kde.StatusNotifierWatcher",
+    "--talk-name=org.freedesktop.Flatpak"
```

Comment:
You have to ask an exception for this https://docs.flathub.org/docs/for-app-authors/linter#exceptions

Error code is https://docs.flathub.org/docs/for-app-authors/linter#finish-args-flatpak-spawn-access

Reply:
Pull request sent

https://github.com/flathub-infra/flatpak-builder-lint/pull/480

Reply:
You can start the build by commenting `bot, build io.github.mhogomchungu.sirikali` once the exception is up https://docs.flathub.org/docs/for-app-authors/linter#check-currently-published-exceptions